### ISSUE TITLE
API giving incomplete results.

### ISSUE THREAD

**Issue by harshitmanek** [MaayanLab/x2k_web#11]
Tried calling the api, it gives incomplete results for both the G2N and X2K. It returns an empty interactions table in case of G2N and no intermediate proteins in case of X2K. The result of the API call cannot be used to build the X2K network map. Please fix this.

**Comment by AviMaayan** [comment]
FYI


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Avi Ma’ayan, PhD
Professor, Department of Pharmacological Sciences
Director, Mount Sinai Center for Bioinformatics
Icahn School of Medicine at Mount Sinai
New York, NY 10029
(212) 241-1153
Lab: http://www.mssm.edu/labs/maayan
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
________________________________

**Comment by maxim-k** [comment]
> Tried calling the api, it gives incomplete results for both the G2N and X2K. It returns an empty interactions table in case of G2N and no intermediate proteins in case of X2K. The result of the API call cannot be used to build the X2K network map. Please fix this.

Could you provide your gene list and parameters, so we can reproduce it?
API section shows example of code to access API. In example we show only top five results using Panda's function `head()`. To access full table you can just remove lines with `head()`. To access result in JSON format you can use `x2k_results['X2K']` for X2K and `x2k_results['G2N']` for G2N.

**Comment by harshitmanek** [comment]
I used the same gene list and parameters which are there in the example. Also I ran the same python code in an ipython notebook and it returned an empty data structure for interactions in the **x2k_results['G2N']** and no intermediate genes in the **x2k_results['X2K']** . Also the p-values returned in the api for both CheA and KEA, are not the same as shown in the webtool. I need to build the X2K network map as shown in the webtool, but am unable to, due to incomplete data that I am getting through the api.
